### PR TITLE
Wercker support

### PR DESCRIPTION
**Pull request by mgi166**
Build in wercker with `"{BUNDLE_UPDATE: true}"`.

screenshot_2017-03-29_8_13_42

### CODE REVIEW

**Comment by masutaka** [comment]
Thanks! I'll check later.

File: ci-bundle-update.rb [hunk 1]
```diff
@@ -1,43 +1,123 @@
...
+  class Wercker < Base
+    include HTTParty
 
-  attr_reader :exec_days
+    base_uri 'https://app.wercker.com/api/v3'
+
+    def initialize(wercker_token, exec_days)
+      super
+
+      @wercker_token = wercker_token
```
Comment (masutaka) [inline ci-bundle-update.rb:60]: `@wercker_token` seems not to use. :) Fixed by 4f07d8b4.

File: ci-bundle-update.rb [hunk 2]
```diff
@@ -1,43 +1,123 @@
...
-    if skip?
-      puts "This build was skipped for $EXEC_DAYS (#{exec_days})"
-      return
+  class CircleCi < Base
+    def initialize(circleci_token, exec_days)
+      super
+
+      ::CircleCi.configure do |config|
+        config.token = circleci_token
+      end
```
Comment (masutaka) [inline ci-bundle-update.rb:30]: I believe it should decrease object dependency. FIxed by 5c75bb56.

**Review by masutaka — COMMENTED** [review]
I've added these commits 25704d89...b221c16e.

Thanks for your PR! :pray:

**Comment by mgi166** [comment]
Thanks!

**Pull request merged**